firmware/raspberry_pi4_network_boot_beta.md

```diff
@@ -0,0 +1,119 @@
+# Raspberry Pi4 network boot (BETA)
+The page describes how to install and configure network boot for the Raspberry Pi4 Model B.
+
+**This is an early beta test and has only been tested on a small number of networks. Check with your network administrator before enabling network boot.**
+
+This documentation will be merged into the main bootloader documents after the beta test is complete.
+
+## Server configuration
+Network boot requires a TFTP and NFS server to be configured.  See [Network boot server tutorial](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bootmodes/net_tutorial.md)
+
+Additional notes:-
+* The VC firmware (start4.elf) must be updated to the latest rpi-update version in order to get the Pi4 network drivers.
+* The MAC address on the Pi4 is programmed at manufacture and is not derived from the serial number.
+
+```
+# mac adddress (ip addr) - it should start with dc:a6:3
```

Comment:
Unless you're got another OUI in the pipeline that the IEEE list (http://standards-oui.ieee.org/oui.txt) doesn't know about, that should have a 2 on the end, i.e. dc:a6:32